src/library/basetracktablemodel.cpp

```diff
@@ -166,6 +167,10 @@ void BaseTrackTableModel::initHeaderProperties() {
             ColumnCache::COLUMN_LIBRARYTABLE_DATETIMEADDED,
             tr("Date Added"),
             defaultColumnWidth() * 3);
+    setHeaderProperties(
+            ColumnCache::COLUMN_LIBRARYTABLE_LAST_PLAYED_AT,
+            tr("Last Time Played"),
```

Comment:
How about just "Last Played" for the column header? Otherwise you might just see "Last Time...".

Reply:
done